Kernel_23/include/CGAL/Kernel/function_objects.h

```diff
@@ -275,6 +275,40 @@ namespace CommonKernelFunctors {
                                 * sq_length(ba)*sq_length(bc));
       }
     }
+
+    Comparison_result
+    operator()(const Point_3& a1, const Point_3& b1, const Point_3& c1,
```

Comment:
You must update the `Kernel` concept (https://doc.cgal.org/latest/Kernel_23/classKernel_1_1CompareAngle__3.html specifically), to reflect this addition to the model.